server/src/models/database/recipe.ts

```diff
@@ -1,15 +1,24 @@
 import { IsNotEmpty, IsNumber, Length } from "class-validator";
+import { ValidationGroup } from "./../enums/validation-group";
 import { FoodItem, RecipeStep, SocoboUser } from "./../index";
 
 export class Recipe {
 
   public fields: Map<string, Function>;
 
   public id: number;
-  @IsNotEmpty()
-  @Length(1, 50)
+
+  @IsNotEmpty({
+    groups: [ ValidationGroup.RECIPE ]
```

Comment:
Is it possible to simply use non validation group if there is only one possible as with the recipe (I cannot really think of another use case for this model) or do we need to always add a validation group?